apis/nucleus/src/components/listbox/components/__tests__/list-box-row-column.spec.jsx

```diff
@@ -834,6 +834,30 @@ describe('<ListBoxRowColumn />', () => {
       await testRenderer.unmount();
     });
 
+    it('should get correct text alignment', async () => {
+      const index = 0;
+      const style = {};
+      const textAlign = 'right';
+      const data = {
+        onMouseDown: sandbox.spy(),
+        onMouseUp: sandbox.spy(),
+        onMouseEnter: sandbox.spy(),
+        onClick: sandbox.spy(),
```

Comment:
Check if we really need to provide these events (since we are not testing them in this test case). Otherwise remove them.